projects/components/src/multi-select/multi-select.component.test.ts

```diff
@@ -138,6 +138,27 @@ describe('Multi Select Component', () => {
     });
   }));
 
+  test('should block prevent default when checkbox is clicked', fakeAsync(() => {
+    spectator = hostFactory(
+      `
+    <ht-multi-select [selected]="selected">
+      <ht-select-option *ngFor="let option of options" [label]="option.label" [value]="option.value">
+      </ht-select-option>
+    </ht-multi-select>`,
+      {
+        hostProps: {
+          options: selectionOptions,
+          selected: []
+        }
+      }
+    );
+
+    spectator.tick();
+    spectator.click('.trigger-content');
+    const selectedCheckboxElement = spectator.queryAll('ht-checkbox', { root: true })[0];
+    expect(spectator.dispatchFakeEvent(selectedCheckboxElement, 'click', true).defaultPrevented).toBe(true);
```

Comment:
This part looks good, but can we add a second validation here that `selectedChange` emits similar to the "should notify and update selection when selection is changed" test - we want to both make sure we're blocking this, but still processing the selection.

Reply:
done. Thanks!